src/main/java/org/jetbrains/bsp/bazel/server/service/BuildServerService.java

```diff
@@ -220,7 +231,16 @@ public void onBuildExit() {
 
   public Either<ResponseError, ResourcesResult> buildTargetResources(
       ResourcesParams resourcesParams) {
-    Build.QueryResult query = queryResolver.getQuery("query", "--output=proto", "//...");
+    BazelProcessResult bazelProcessResult =
+        bazelRunner
+            .commandBuilder()
+            .query()
+            .withFlag(BazelRunnerFlag.OUTPUT_PROTO)
+            .withArgument("//...")
```

Comment:
Maybe we should extract it as a constant?

Reply:
this call should be changed, but not in this PR